### PR TITLE
Extend-Discovery: Fix azure lowercase issues

### PR DESCRIPTION
**Pull request by lebogg**

### CODE REVIEW

File: service/discovery/azure/azure.go [hunk 1]
```diff
@@ -505,7 +505,7 @@ func resourceGroupID(ID *string) voc.ResourceID {
 		return ""
 	}
 
-	id := strings.Join(s[:5], "/")
+	id := strings.ToLower(strings.Join(s[:5], "/"))
```
Comment (anatheka) [inline service/discovery/azure/azure.go:508]: You also could use the function resourceID() for the string in line 501.
`s := strings.Split(resourceID(ID), "/")`

File: service/discovery/azure/compute.go [hunk 2]
```diff
@@ -870,6 +870,8 @@ func addSecretUsages(webAppID *string, settings armappservice.WebAppsClientListA
 		if s := util.Deref(v); strings.Contains(s, "@Microsoft.KeyVault") {
 			sURI := getSecretURI(s)
 			secretUsage[sURI] = append(secretUsage[sURI], util.Deref(webAppID))
+			sUsages := secretUsage
+			_ = sUsages
```
Comment (anatheka) [inline service/discovery/azure/compute.go:874]:
```suggestion
```

File: service/discovery/azure/storage.go [hunk 3]
```diff
@@ -771,10 +771,12 @@ func (d *azureStorageDiscovery) handleTableStorage(account *armstorage.Account,
 	if err != nil {
 		return nil, fmt.Errorf("could not get object storage properties for the atRestEncryption: %w", err)
 	}
+	backupOFF := backupOf
+	_ = backupOFF
```
Comment (anatheka) [inline service/discovery/azure/storage.go:775]:
```suggestion
	_ := backupOf
```

**Review by anatheka — APPROVED** [review]
Some minor comments.

File: service/discovery/azure/azure.go (hunk 1 above)
Comment (lebogg) [reply service/discovery/azure/azure.go:508]: That's right, i think 👍 I just used the code of main

**Pull request merged**